<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SamplerControls</class>
 <widget class="QWidget" name="SamplerControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>543</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Sample Warper Controls</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: yellow;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>241</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: &quot;Comic Sans MS&quot;;
	color: magenta;
}</string>
   </property>
   <property name="text">
    <string>Sample Warping Controls</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: &quot;Comic Sans MS&quot;;
	color: blue;
;
}</string>
   </property>
   <property name="text">
    <string>Sample Count</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="numSamplesSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #ff85d0, stop:1 #ff3fb4);
    border: 1px solid #5c5c5c;
    width: 10px;
    margin: -7px 0; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
    border-radius: 3px;
}

QSlider::groove:horizontal {
    border: 1px solid #999999;
    height: 8px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #ff85d0, stop:1 #cb67a4);
    margin: 2px 0;
}</string>
   </property>
   <property name="minimum">
    <number>4</number>
   </property>
   <property name="maximum">
    <number>16</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>2</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="numSamplesSpinBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox{
	background-color: rgb(226, 153, 186);
    border-radius: 5px;
}
QSpinBox::down-button{
	color: pink;
    border-radius: 5px;
}
QSpinBox::up-button{
	color: black;
    border-radius: 5px;
}
</string>
   </property>
   <property name="minimum">
    <number>16</number>
   </property>
   <property name="maximum">
    <number>1048576</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>1024</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    border: 1px solid rgb(255, 85, 255);
    border-radius: 5px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-width: 1px;
    border-left-color: rgb(255, 85, 255);
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 5px; /* same radius as the QComboBox */
    border-bottom-right-radius: 5px;
}
QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}

QComboBox QAbstractItemView {
    border: 2px solid rgb(255, 170, 255);
    selection-background-color: rgb(255, 170, 255);
	font: &quot;Comic Sans MS&quot;;
    border-radius: 5px;
}</string>
   </property>
   <item>
    <property name="text">
     <string>Random</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Grid</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Stratified</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: &quot;Comic Sans MS&quot;;
	color: blue;
;
}</string>
   </property>
   <property name="text">
    <string>Sample Distribution</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: &quot;Comic Sans MS&quot;;
	color: blue;
;
}</string>
   </property>
   <property name="text">
    <string>Warping Method</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    border: 1px solid rgb(255, 85, 255);
    border-radius: 5px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-width: 1px;
    border-left-color: rgb(255, 85, 255);
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 5px; /* same radius as the QComboBox */
    border-bottom-right-radius: 5px;
}
QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}

QComboBox QAbstractItemView {
    border: 2px solid rgb(255, 170, 255);
    selection-background-color: rgb(255, 170, 255);
	font: &quot;Comic Sans MS&quot;;
    border-radius: 5px;
}</string>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Disc (Uniform)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Disc (Concentric)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sphere</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sphere Cap</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hemisphere (Uniform)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hemisphere (Cosine Weighted)</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>119</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator{
    border: 1px solid rgb(255, 85, 255);
    border-radius: 5px;
    padding: 1px 1px 1px 3px;
}

QCheckBox::indicator:unchecked {
    background-color: yellow;
}

QCheckBox::indicator:checked {
    background-color: rgb(255, 170, 255)
}</string>
   </property>
   <property name="text">
    <string>Display Grid</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>241</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: &quot;Comic Sans MS&quot;;
	color: magenta;
}</string>
   </property>
   <property name="text">
    <string>PDF Integral Results</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: &quot;Comic Sans MS&quot;;
	color: blue;
;
}</string>
   </property>
   <property name="text">
    <string>Disc (Uniform)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: &quot;Comic Sans MS&quot;;
	color: blue;
;
}</string>
   </property>
   <property name="text">
    <string>Disc (Concentric)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: &quot;Comic Sans MS&quot;;
	color: blue;
;
}</string>
   </property>
   <property name="text">
    <string>Sphere</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: &quot;Comic Sans MS&quot;;
	color: blue;
;
}</string>
   </property>
   <property name="text">
    <string>Sphere Cap</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: &quot;Comic Sans MS&quot;;
	color: blue;
;
}</string>
   </property>
   <property name="text">
    <string>Hemisphere (Uniform)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: &quot;Comic Sans MS&quot;;
	color: blue;
;
}</string>
   </property>
   <property name="text">
    <string>Hemisphere (Cosine)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>350</y>
     <width>16</width>
     <height>121</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>490</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 2px solid rgb(85, 255, 255);
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(255, 170, 255), stop: 1 rgb(255, 85, 255));
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgb(255, 85, 255), stop: 1 rgb(255, 170, 255));
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
   </property>
   <property name="text">
    <string>Compute PDFs</string>
   </property>
  </widget>
  <widget class="QLabel" name="discUnifPDFLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>350</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: &quot;Comic Sans MS&quot;;
	color: blue;
;
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="discConcPDFLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: &quot;Comic Sans MS&quot;;
	color: blue;
;
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="spherePDFLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>390</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: &quot;Comic Sans MS&quot;;
	color: blue;
;
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="sphereCapPDFLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>410</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: &quot;Comic Sans MS&quot;;
	color: blue;
;
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="hemiUnifPDFLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>430</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: &quot;Comic Sans MS&quot;;
	color: blue;
;
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="hemiCosPDFLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>450</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: &quot;Comic Sans MS&quot;;
	color: blue;
;
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="thetaMaxSpinBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox{
	background-color: rgb(226, 153, 186);
    border-radius: 5px;
}
QSpinBox::down-button{
	color: pink;
    border-radius: 5px;
}
QSpinBox::up-button{
	color: black;
    border-radius: 5px;
}
</string>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: &quot;Comic Sans MS&quot;;
	color: blue;
;
}</string>
   </property>
   <property name="text">
    <string>Min Theta</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>numSamplesSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SamplerControls</receiver>
   <slot>slot_setSamples(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SamplerControls</receiver>
   <slot>slot_setSampleMode(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_2</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SamplerControls</receiver>
   <slot>slot_setWarpMethod(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked()</signal>
   <receiver>SamplerControls</receiver>
   <slot>slot_setDisplayGrid()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SamplerControls</receiver>
   <slot>slot_computePDFs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>503</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>513</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>thetaMaxSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SamplerControls</receiver>
   <slot>slot_updateThetaMax(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>528</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_setSamples(int)</slot>
  <slot>slot_setSampleMode(int)</slot>
  <slot>slot_setWarpMethod(int)</slot>
  <slot>slot_setDisplayGrid()</slot>
  <slot>slot_computePDFs()</slot>
  <slot>slot_updateThetaMax(int)</slot>
 </slots>
</ui>
